bin/oscapd-evaluate

```diff
@@ -176,12 +176,11 @@ def cli_scan(args, config):
                                 args.xccdf_id, args.profile
                             )
                         except ProfileSuffixMatchError as e:
-                            logging.error(
-                                "Failed to find profile matching suffix '%s' "
-                                "in profiles applicable to target '%s': %s"
-                                % (args.profile, target, str(e))
+                            msg = (
+                                "Failed to pick a profile for scanning '{}': {}"
+                                .format(target, str(e))
```

Comment:
I agree that the message is pretty technical but IMO the new message doesn't say enough. Pick a profile by what? If I imagine that I don't know about the suffix matching I wouldn't be able to figure this out without looking at the source code.

Reply:
Or does the exception message have all the info?

Reply:
Yes, the second parts of messages are at `openscap_daemon/evaluation_spec.py:614` and I think that they provide sufficient user feedback. I am not sure whether to inform about the target, because it looks like an internal variable.
If I specify a wrong profile, I now get
```
ERROR:Failed to pick a profile for scanning 'chroot:///scanin/98a88a8b722a71835dd761c88451c681a8f1bc6e577f90d4dc8b234100bd4861': No profile with suffix nothing
```